node_helper.js

```diff
@@ -10,6 +10,7 @@
 const NodeHelper = require('node_helper');
 var async = require('async');
 var exec = require('child_process').exec;
+var request = require('request');
```

Comment:
Is the request dependency installed with MM or is it necessary to install it via npm install?
If via npm please add it to the `package.json` file.